_posts/2020-10-07-news.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: The status of the development
+---
+
+# Weekly git tagging
+
+- [#2655](https://github.com/universal-ctags/ctags/pull/2655)
+- [#1038](https://github.com/universal-ctags/ctags/issues/1038)
+
+@masatake is implementing weekly automatic git tagging run at GitHub action.
```

Comment:
```suggestion
@masatake is implementing weekly automatic git tagging run at GitHub Actions.
```
`GitHub Actions` is the correct name of the service.

Reply:
Thank you.